src/algos/simulation.jl

```diff
@@ -20,49 +22,102 @@ function simulate(model::AbstractNumericModel, dr::AbstractDecisionRule,
     nsx = nx+ns
 
     # TODO: talk to Pablo and make a decision about this
-    s_simul = Array(Float64, n_exp, ns, horizon)
-    x_simul = Array(Float64, n_exp, nx, horizon)
-    for i in 1:n_exp
+    s_simul = Array(Float64, N, ns, T)
+    x_simul = Array(Float64, N, nx, T)
+    for i in 1:N
       s_simul[i, :, 1] = s0
       x_simul[i, :, 1] = x0
     end
-    # NOTE: this will be empty if ny is zero. That's ok. Our call to `cat`  #       below will work either way  y_simul = Array(Float64, n_exp, ny, horizon)
+    # NOTE: this will be empty if ny is zero. That's ok. Our call to `cat`  #       below will work either way  y_simul = Array(Float64, N, ny, horizon)
 
-    for t in 1:horizon
+    for t in 1:T
         s = copy(view(s_simul, :, :, t))
```

Comment:
Here I think we should remove the `copy` as it defeats the purpose of calling `view`